webapp/src/pages/UserStatsPage/UserStatsPage.css

```diff
@@ -0,0 +1,49 @@
+.stats-page {
+  height: 100vh;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  display: grid;
+  grid-template-columns: repeat(var(--column-number), 1fr);
+  grid-template-rows: auto 1fr;
+  gap: 3rem var(--column-gap);
+  background-color: var(--color-background);
+  color: var(--color-light);
+  max-height: available;
+  overflow: clip;
+}
+
+.stats-page .header {
+  grid-column: 1/-1;
+  grid-row: 1/2;
+  align-self: center;
+}
+
+.stats-page .graphs {
+  grid-column: 1/-1;
+  grid-row: 2/-1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  gap: var(--row-gap) var(--column-gap);
+}
+
+.stats-page .graphs .line-graph-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.stats-page .graphs .line-graph-wrapper canvas {
+  min-height: 18rem;
+  max-width: 60vw;
+}
+
+.stats-page .graphs .pie-chart-wrapper {
```

Comment:
the style inside the canvas responds to css changes only on reload, no hot reloading. I.e. The component will create element.style based on the style sheet and only update it on reload.